### PR TITLE
Initial commit for Ukrainian localization

### PR DESCRIPTION
**Pull request by MaxymVlasov**

### CODE REVIEW

**Comment by Potapy4** [comment]
Я також можу допомогти з ревью PR'ов 🙂

File: README-uk.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+* [Керівництво зі стилю оформлення документації](http://kubernetes.io/docs/contribute/style/style-guide/)
+* [Переклад документації Kubernetes іншими мовами](https://kubernetes.io/docs/contribute/localization/)
+
+## Запуск сайту локально за допомогою Docker
+
+Для локального запуску вебсайту Kubernetes рекомендовано запустити спеціальний [Docker](https://docker.com)-образ, що містить генератор статичних сайтів [Hugo](https://gohugo.io).
+
+> Якщо ви працюєте під Windows, вам знадобиться ще декілька інструментів, які можна встановити за допомогою [Chocolatey](https://chocolatey.org). `choco install make`
+
+> Якщо ви вважаєте кращим запустити вебсайт локально без використання Docker, дивіться пункт нижче [Запуск сайту локально за допомогою Hugo](#running-the-site-locally-using-hugo).
```
Comment (sftim) [inline README-uk.md]: nit: `#running-the-site-locally-using-hugo` does not match generated `id` attribute which I think will be `запуск-сайту-локально-з-допомогою-hugo`

Comment (sftim) [reply README-uk.md]: Should that be “Запуск сайту локально **за** допомогою Hugo” and `запуск-сайту-локально-за-допомогою-hugo`?

**Comment by sftim** [comment]
@Potapy4 feel free to /lgtm if it does indeed look good to you.

File: README-uk.md (hunk 1 above)
Comment (MaxymVlasov) [reply README-uk.md]: Yes, thanks

**Comment by MaxymVlasov** [comment]
@Potapy4, @sftim would you like to be added to `sig-docs-uk-reviews` and participate in reviews future PRs?

**Comment by Potapy4** [comment]
> would you like to be added to `sig-docs-uk-reviews` and participate in reviews future PRs?

yep, I'm on it 🙂

/lgtm

**Comment by MaxymVlasov** [comment]
Cool! If you'd like, you can also join `#kubernetes-docs-uk` Slack channel

**Comment by butuzov** [comment]
і понеслось... !

**Comment by sftim** [comment]
Re: https://github.com/kubernetes/website/pull/18569#issuecomment-573011290

**Comment by sftim** [comment]
Would you like a new branch created for Ukrainian localization, as per [branching strategy](https://kubernetes.io/docs/contribute/localization/#branching-strategy)? If so, I'm sure that can happen.

**Comment by MaxymVlasov** [comment]
@sftim yes, separate branch can be helpful, I will ask about it Roy in nearest future.
Also, looks like all OK. For go forward and merge PR, we need `/approve` from your side?

**Comment by sftim** [comment]
It's fine to work on a branch and build a series of changes that together meet the minimum requirements.

At the moment, https://deploy-preview-18569--kubernetes-io-master-staging.netlify.com/uk/ does not look ready for merge.

I will ask on Slack about getting a new branch.

**Comment by sftim** [comment]
@kubernetes/sig-docs-l10n-admins, please create a new branch for Ukrainian / Українська (`uk`) localization.

**Comment by anastyakulyk** [comment]
Re: https://github.com/kubernetes/website/pull/18569#issuecomment-573646357
@sftim many thanks for your feedback! Do I understand it right that to have our localization PR merged we also need to translate the content from the front page? Could you please specify where can we find those files? Because they are not mentioned in minimum content requirements https://kubernetes.io/docs/contribute/localization/#minimum-required-content

**Comment by sftim** [comment]
@anastyakulyk please see the table at https://kubernetes.io/docs/contribute/localization/#translating-content
If you work on a localization branch you need all of those minimum things before the branch can merge into _master_.

…and there **is** a localization branch now, so you can open pull requests that target [_dev-1.17-uk.1_](https://github.com/kubernetes/website/tree/dev-1.17-uk.1)
@MaxymVlasov would you be OK to either change the target branch for this PR, or to open a new one that targets _dev-1.17-uk.1_?

**Comment by anastyakulyk** [comment]
https://github.com/kubernetes/website/pull/18569#issuecomment-573722995
@sftim thanks, got it!

**Comment by MaxymVlasov** [comment]
@sftim thanks. I changed base branch to `dev-1.17-uk.1`. 

Also, I have a little question about review and PR flow. We need translate all described in [translating content](https://kubernetes.io/docs/contribute/localization/#translating-content) by this PR or can create many non-big PR's which  after `/lgtm` can be merged into `dev-1.17-uk.1` branch?

**Comment by sftim** [comment]
This PR is enough to kick things off on the branch for Ukrainian localization.

When you're ready, you can take everything from _dev-1.17-uk.1_, squash it, and propose a PR to merge into _master_.

/approve

**Comment by OleksaBaida** [comment]
Hello, guys

is there any TL;DR like howto/page/guide for wannabes who looking how to participate in the translation?

**Comment by butuzov** [comment]
You can join https://slack.kubernetes.io/  chan #kubernetes-docs-uk

**Comment by MaxymVlasov** [comment]
@OleksaBaida please check issue below, it have minimum for start contributing
https://github.com/kubernetes/website/issues/20786

**Comment by OleksaBaida** [comment]
thanks @MaxymVlasov . Looks just what I was looking for. Great job!

**Pull request merged**